applications/API/migrations/0006_insultreview_insult.py

```
@@ -0,0 +1,25 @@
+        migrations.AddField(
+            model_name="insultreview",
+            name="insult",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="reports",
+                to="API.insult",
+            ),
+        ),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)